go.mod

```diff
@@ -1,71 +1,74 @@
 module github.com/k8snetworkplumbingwg/sriov-network-operator
 
-go 1.22.4
+go 1.23.0
+
+toolchain go1.23.4
```

Comment:
do we need toolchain directive?

https://go.dev/ref/mod#go-mod-file-toolchain

Reply:
it gets added when I run `go mod tidy` I didn't add it